samples/sdl_ttf/main.c

```diff
@@ -0,0 +1,145 @@
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/xbox.h>
+#include <hal/video.h>
+#include <SDL.h>
+#include <SDL_ttf.h>
+#include <stdbool.h>
+
+const extern int SCREEN_WIDTH;
+const extern int SCREEN_HEIGHT;
+
+// SDL requires this symbol to exist
+void _exit() {
+}
+
+void main() {
+  int init_ret;
+  bool initialized_pbkit = false;
+  bool initialized_SDL   = false;
+  bool initialized_TTF   = false;
+
+  init_ret = pb_init();
+  if (init_ret != 0) {
+    debugPrint("pb_init() failed!");
+    XSleep(2000);
+    return;
+  }
+  initialized_pbkit = true;
+  pb_show_debug_screen();
+  
+  init_ret = SDL_VideoInit(NULL);
+  if (init_ret != 0) {
+    debugPrint("SDL_VideoInit failed: %s", SDL_GetError());
+    XSleep(2000);
+    goto cleanup;
+    return;
+  }
+  initialized_SDL = true;
+
+  init_ret = TTF_Init();
+  if (init_ret != 0) {
+    debugPrint("TTF_Init failed: %s", TTF_GetError());
+    XSleep(2000);
+    goto cleanup;
+    return;
+  }
+  initialized_TTF = true;
```

Comment:
I just realized you could just use `ttf_initialization = TTF_Init()` and check for 0 in cleanup.
You'd declare it as `int ttf_initialization = -1;` or something to signal that it wasn't initialized yet.

Reply:
That's a very good idea!